client/src/templates/Challenges/projects/frontend/Show.tsx

```diff
@@ -168,7 +168,9 @@ class Project extends Component<ProjectProps> {
         <LearnLayout>
           <Helmet
             title={`${blockNameTitle} | ${t('learn.learn')} | freeCodeCamp.org`}
-          />
+          >
+            <style>{'#learn-app-wrapper{display:initial;}'}</style>
```

Comment:
Sorry, @ahmadabdolsaheb, but this style also needs to be overridden only when the `Project` component is mounted, so in order to do that, we need to leave that line alone in the `Project` component's Helmet.

Reply:
I don't seem to be able to reproduce this on my side:
<img width="1254" alt="Screen Shot 2022-11-10 at 10 44 06 AM" src="https://user-images.githubusercontent.com/4591597/201030058-ab722c35-d908-4117-8cca-21426c25a742.png">

Reply:
[@ahmadabdolsaheb](https://github.com/ahmadabdolsaheb),

Thanks for checking it, and please make the page narrower (or press Toggle device toolbar button).
Without `display: initial;`, The instruction is stretched beyond the viewport like below image.

![stretched_instruction](https://user-images.githubusercontent.com/44451585/201044573-f15c5d81-430e-475d-95b4-24af6062fa00.png)

Reply:
@mmatsumoto1026 True, but that issue happened because you have added multiple `display: flex` for three layers. So either create Grid layout, or use specific selector with `:has`, `:is` and `:not` pseudo-classes to target the area in which scrollbar is visible

In the main site, there is no issue

Reply:
Thanks for your advice, @Sboonny.

It seems that the code can be refactored with using `:has` and `:not` like below to solve some issues.
After test, I will push the fixed code.

```css
#learn-app-wrapper:has(.container) {
  display: initial;
}
```
```css
.page-wrapper:not(:has(.site-footer)) .default-layout {
  min-height: 0;
}
```

Reply:
`:has` seems to be not supported in Firefox, so `:only-child` pseudo class was used to solve `min-height: 0;` issue.

Reply:
True has isn't supported, but we are using postcss which should take care of this issue.

https://github.com/freeCodeCamp/freeCodeCamp/blob/main/client/postcss.config.js